### PR TITLE
feat(helm): add logging-operator flows

### PR DESCRIPTION
**Pull request by genofire**
### Summary
@Rash419 it would be more beautiful, if collabora would start use machine-readable logs (like [logfmt](https://www.brandur.org/logfmt) or json)

### TODO

- validate if it works with
  - [x] dynConfig-nginx
  - [x] dynConfig-upload

### Checklist

- [x] Code is properly formatted
- [x] All commits have Change-Id
- [x] Documentation (manuals or wiki) has been updated or is not required:
  - README update with new Logging section, here rendered: https://github.com/genofire/collabora-online/blob/feat/helm-logging/kubernetes/helm/collabora-online/README.md#kubernetes-cluster-logging

### CODE REVIEW

**Comment by genofire** [comment]
Ping @Rash419

**Review by Rash419 — CHANGES_REQUESTED** [review]

File: kubernetes/helm/collabora-online/templates/flow.yaml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        hash_value_field: "collabora"
+        remove_key_name_field: true
+        reserve_data: true
+        parse:
+          type: "multi_format"
+          patterns:
+            - format: "regexp"
+              # regex for wopihost and wopidoc
+              # wsd-00001-00092 2023-04-20 15:34:56.729798 +0000 [ docbroker_001 ] ERR  Failed to add session to [https://nextcloud.example.org:443/collabora/lptFile123.odt] with URI [https://nextcloud.example.org/collabora/lptFile123.odt?access_token=1&access_token_ttl=0&permission=edit]: No acceptable WOPI hosts found matching the target host [nextcloud.example.org] in config.| wsd/DocumentBroker.cpp:2251
+              # wsd-00001-00092 2023-04-20 15:34:56.729826 +0000 [ docbroker_001 ] ERR  Unauthorized Request while starting session on https://nextcloud.example.org:443/collabora/lptFile123.odt for socket #33. Terminating connection. Error: No acceptable WOPI hosts found matching the target host [nextcloud.example.org] in config.| wsd/COOLWSD.cpp:4727
```
[the remark this answers is not in the document]
Comment (genofire) [reply kubernetes/helm/collabora-online/templates/flow.yaml]: done

**Review by Rash419 — APPROVED** [review]
Looks good

**Comment by Rash419** [comment]
Merging this. There is no way it can affect the cypress/unit tests

**Pull request merged**